packages/core-mobile/app/store/migrations.ts

```diff
@@ -302,5 +306,20 @@ export const migrations = {
         disabledLastTransactedChainIds: []
       }
     }
+  },
+  19: (state: any) => {
+    const isDeveloperMode = state.settings.advanced.developerMode
+    // in the mobile next-gen app, we don't have the concept of active network anymore,
+    // we will default active network to avalanche c-chain until we remove active network
+    // everywhere in the app
```

Comment:
let's switch the developer mode to false if it is on and do only 43114. I don't think our logic will work with 43113 (fuji)